posthog/api/test/test_event.py

```diff
@@ -264,14 +265,23 @@ def test_before_and_after(self):
             self.assertEqual(response["results"][0]["id"], event2.pk)
 
         def test_pagination(self):
-            events = []
             person_factory(team=self.team, distinct_ids=["1"])
-            for index in range(0, 150):
-                event_factory(team=self.team, event="some event", distinct_id="1")
+            for _ in range(0, 150):
+                event_factory(
+                    team=self.team, event="some event", distinct_id="1", timestamp=datetime(2020, 1, 1, 12, 0, 0, _)
+                )
```

Comment:
To be super thorough, it might make sense to test while incrementing on days so it can test the two param order by we now use

Reply:
Added, that could have uncovered what was going on!